packages/react/src/Accordion/AccordionSection.tsx

```diff
@@ -46,23 +47,13 @@ export const AccordionSection = forwardRef(
             {label}
           </button>
         </HeadingX>
-        {section ? (
-          <section
-            id={panelId}
-            aria-labelledby={buttonId}
-            className={clsx('ams-accordion__panel', { 'ams-accordion__panel--expanded': isExpanded })}
-          >
-            {children}
-          </section>
-        ) : (
-          <div
-            id={panelId}
-            aria-labelledby={buttonId}
-            className={clsx('ams-accordion__panel', { 'ams-accordion__panel--expanded': isExpanded })}
-          >
-            {children}
-          </div>
-        )}
+        <Tag
+          id={panelId}
+          aria-labelledby={buttonId}
+          className={clsx('ams-accordion__panel', { 'ams-accordion__panel--expanded': isExpanded })}
```

Comment:
These can be sorted as well.

```suggestion
          aria-labelledby={buttonId}
          className={clsx('ams-accordion__panel', { 'ams-accordion__panel--expanded': isExpanded })}
          id={panelId}
```